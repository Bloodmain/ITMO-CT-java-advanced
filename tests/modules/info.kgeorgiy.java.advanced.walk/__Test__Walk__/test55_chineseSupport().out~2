d38e6317c26951c29c886dfe40bb79876d6073c2 __Test__Walk__/test55_chineseSupport()/程國國問持支支國中中序中請序支持你你程國國你支的持序的中的的
909c42f0785e2a860deb959aba102d5a3b58288f __Test__Walk__/test55_chineseSupport()/的中問程持程問持支序你中的持中程你中請的問支程問中你請程你你
21c552200e6607916ecf34bd2c15cb65a636ff07 __Test__Walk__/test55_chineseSupport()/請持支你問持國支國問支持國中問的的中程的支程你支問序支請的序
e44ef817d8008529ce4504cddc4acffaa2ac1a93 __Test__Walk__/test55_chineseSupport()/中中的持問的的中請持程程程國請序的你的請你請的支持你國持中序
ea0ad743c5259a29835ecf550dd1717e31ab8cad __Test__Walk__/test55_chineseSupport()/的程程序序序持支持國你序你請請序請序持問持中國支國程中你中的
44e24a38f9e9e2fc05cda3118dd6070516a63842 __Test__Walk__/test55_chineseSupport()/的問支國請持問支序序問國請你的序持問你中持國中中你問請序問持
465e15cff73ef8f9d26408251152f3e2920529b2 __Test__Walk__/test55_chineseSupport()/持序請支中程國程問的國你持程支問請支的持中中持問問持支程問序
fdc1d743612a73db6d7076df5a9795b0da062c97 __Test__Walk__/test55_chineseSupport()/問支中問支中的持持持程你持持的中問你序問程中支的問國程國序請
8a5a886deea5025278ab390b9879fd0c496e4672 __Test__Walk__/test55_chineseSupport()/中請問問支中你問的持的的國序問序你的問的國你問持請序中的的請
6ca6a78c50ce6d0814dc52dca34fa23867f866a9 __Test__Walk__/test55_chineseSupport()/中持的國你你請程支國中你你問程的請的中持請持你請程程支中程問
