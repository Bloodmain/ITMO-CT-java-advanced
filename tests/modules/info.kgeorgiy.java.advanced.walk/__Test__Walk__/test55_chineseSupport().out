8e92e51b __Test__Walk__/test55_chineseSupport()/程國國問持支支國中中序中請序支持你你程國國你支的持序的中的的
611b3841 __Test__Walk__/test55_chineseSupport()/的中問程持程問持支序你中的持中程你中請的問支程問中你請程你你
55538cc5 __Test__Walk__/test55_chineseSupport()/請持支你問持國支國問支持國中問的的中程的支程你支問序支請的序
235ad359 __Test__Walk__/test55_chineseSupport()/中中的持問的的中請持程程程國請序的你的請你請的支持你國持中序
db26632e __Test__Walk__/test55_chineseSupport()/的程程序序序持支持國你序你請請序請序持問持中國支國程中你中的
7d4b7e70 __Test__Walk__/test55_chineseSupport()/的問支國請持問支序序問國請你的序持問你中持國中中你問請序問持
b57543ca __Test__Walk__/test55_chineseSupport()/持序請支中程國程問的國你持程支問請支的持中中持問問持支程問序
396d3551 __Test__Walk__/test55_chineseSupport()/問支中問支中的持持持程你持持的中問你序問程中支的問國程國序請
ded21845 __Test__Walk__/test55_chineseSupport()/中請問問支中你問的持的的國序問序你的問的國你問持請序中的的請
b26d818b __Test__Walk__/test55_chineseSupport()/中持的國你你請程支國中你你問程的請的中持請持你請程程支中程問
